iron-iconset-svg.html

```diff
@@ -63,14 +63,6 @@
       },
 
       /**
-       * Array of fully-qualitifed icon names in the iconset.
-       */
-      iconNames: {
```

Comment:
While this change is fine, it's notable in that it "breaks" the API for this element. What are your thoughts re: versioning?